#!/usr/bin/env python
PACKAGE='cpt_ros'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("alpha",    double_t,    0, "An Integer parameter", 1.6,  0, 10)
gen.add("beta",    double_t,    0, "An Integer parameter", 5.5,  0, 10)
gen.add("c",    double_t,    0, "An Integer parameter", 0.8,  0, 2)

gen.add("alpha_z",    double_t,    0, "An Integer parameter", 5.8,  0, 10)
gen.add("beta_z",    double_t,    0, "An Integer parameter", 7.0,  0, 10)
gen.add("c_z",    double_t,    0, "An Integer parameter", 0.8,  0, 2)
exit(gen.generate(PACKAGE, "cpt_ros", "RMPConfig"))
